axiom/monitors.go

```diff
@@ -28,6 +28,13 @@ const (
 	AboveOrEqual // AboveOrEqual
 )
 
+type Type uint8
+
+const (
+	Threshold  Type = iota // Threshold
```

Comment:
If `Threshold` is not the default, consider adding an `empty`/`unknown` value. Is it?